nemo_curator/utils/file_utils.py

```diff
@@ -18,7 +18,7 @@
 import pathlib
 import warnings
 from functools import partial, reduce
-from typing import List, Optional, Union
+from typing import Dict, List, Optional, Union
```

Comment:
```suggestion
from typing import List, Optional, Union
```
It does not look like this is being used.